contributing/USER_PERSONAS.md

```diff
@@ -0,0 +1,239 @@
+    encompassing and still require a significant amount of proprietary
+    development by their customers to fully integrate into their
+    workflows. For this they are required to provide stable, and robust
+    APIs as well as end-user functionality.
+-   Codebases _may_ be much newer, than the proprietary ones.
+
+Interested in:
+
+-   Stable, robust, documented APIs.
+-   Ways to abstract specific customer requirements.
+-   Reducing the integration cost for customers.
+-   Reducing the number of permutations in the integration matrix.
+-   Facilitating interoperability between tools.
+-   Distributed and scalable systems.
+
+Uses OpenAssetIO to:
+
+-   Minimize the cost of integrating their product into other software
+    and DCC tools.
+
+## Pipeline Technical Director (TD)
+
+> Pipeline: The overarching term used to describe the systems that
+> manage data and work as it moved between artists within media creation
+> facilities.
+
+These developers work as part of an in-house team to solve technical
+challenges that arise during the delivery of a project. They may well
+also be part of a team with the Proprietary DAM/AMS/PT system authors
+described above. Though often more focused towards project-specific
+deliverables, they also contribute to the general maintenance and
+development of the pipeline itself.
+
+When working under the broader remit, and on more immediate solutions,
+their work may tend towards higher-level APIs and scripts over low-level
+frameworks and shared library development. This means that from the
+perspective of OpenAssetIO they may quickly jump from the host API to
+the manager API. E.g, using the host API to quickly build scripts to
+deal with data or asset management tasks. Maybe to support an ad-hoc
+workflow required by a particular task that doesn't fit into the general
+pipeline.
+
+> There is debate as to whether they would use an abstraction such as
+> OpenAssetIO over the first-class native SDK of the pipeline, but as it
+> is a possibility, we should consider it.
+
+-   **OpenAssetIO API alignment:** Manager, Host
+-   **Languages:** Python, Lua, CSS, HTML, \*SQL, GraphQL. Rust has
+    started to pick up momentum in some studios.
+-   **Platforms:** Linux, Windows, macOS
+-   **Frameworks:** Qt, pyside, database technologies
+-   **Codebase age:** From short-lived to long term depending on
+    specific tasks.
+-   **Ability to adopt new language standards:** Mixed, often
+    constrained by technology choices that exist within the pipeline
+    itself, however new solutions may be free to innovate as required to
+    solve a specific need.
+
+Interested in:
+
+-   Simple, easy to us, stable, robust, documented APIs.
+-   Ways to solve a problem quickly.
+-   Interoperability between tools.
+-   Automation.
+-   Availability of tools via common scripting languages.
+
+Uses OpenAssetIO to:
+
+-   Quickly build tools and scripts that interact with asset management
```

Comment:
[question] Is there a use case of bridging multiple asset management systems, e.g. for subcontracting, or combined use of commercial and in-house systems? 

I guess this would be covered by your more general statement, but from my naive perspective some more specific examples would be nice.

Reply:
Great point. Added [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/268/commits/4bd79aaee88d509f9459b4500967d2cfdd989f11)